dev_apps/music-nga/elements/music-rating.js

```diff
@@ -0,0 +1,103 @@
+(function(window) {
+'use strict';
+
+var proto = Object.create(HTMLElement.prototype);
+
+var template =
+`<style scoped>
+  #container {
+    display: flex;
+    flex-flow: row nowrap;
+    width: 100%;
+    height: 100%;
+  }
+  #container > button {
+    background-color: transparent;
+    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAQAAAC0NkA6AAAB2UlEQVR4Ad3XT0vjWhjA4Sfpv9gLQqlahTv7+f6fRcSO2lFHdGFxOtJ6m7Y2d5NFKBpyarKZ50A3hfx4zxso9deIBMk6vuGXtQDtTJB/DZD5KUAsRM8ADPSai5yKQOS0qUjHUM5Qp5nISCQnMmoi0nKs6Fir/siJWFHspO5I7KQ8W0fkSNuutqPaIiVrHonqiwx1faRrWF9kFP5NaGQg8ZnE4KuRSOLQmTJnDiUiJaKMorZefrp6uqpbSaX5Z2qzG2kbyB8tVo+tND+/baKM7/7RnIXLGIkmJcR40KQHYkzdyTQhc2dKDKZuG8hkbk3JI3gxqTmTmXihGGHm2lZdtq7N2I3w6sq7Ory78spHEeZhmZLEnM8iLIxtfMXGDwvKIvxnbG1fa2NvlEdgaWxlHytjS6pESPNpwqdIqRphZStUZkVIpKUnVFcrLHJgH/2wSN8+DhqfJDzSb/66Iol9JKLqkZ7YPmK96pE+1HdhceDal27cWIatPmSStXsXZmYu3FvXP8nWk3PPMpB5du7Jdv9J2joK8kc+2u5kH/NsUUe7WqSvKL+c0gss6leLJMXfydI1y1+FsQW5pFrkTYbUxKW5KuYuTaTIvJX9dSjq6/ojEyZyaF0Sadb/h/OY56tQNmIAAAAASUVORK5CYII=);
```

Comment:
Is there a reason we're embedding the background-image like this? I find it harder to debug when I can't look at the CSS and know what the image should be (as I can if it's just a URL to a file on the disk somewhere).